### PR TITLE
Styled charts and moved additional options text to locales

### PR DESCRIPTION
**Pull request by sergiomarrocoli**
"The bar charts are not right – there is no label on the axis (y-axis is the number of commitments x-axis is the actor).  The chart could be made wider. The figure is now incorrectly labelled – you can remove that text."

- labels the axes
- made chart wider
- removed label text
- moved additional info in manager form options to a yml instead of in the class name

TO DO: still a lot of styling changes to make to the map, so apart from what has been changed here, please ignore other map styling issues
 
Testing:
- Click charts on the home page and check above changes are applied
- go to criteria form and check it displays the options with the additional text

### CODE REVIEW

File: app/assets/stylesheets/components/_map.scss [hunk 1]
```diff
@@ -142,7 +140,7 @@ display: none;
     line-height: 1;
     margin: 0;
     padding: 0 rem-calc(12);
-    min-width: rem-calc(140);
+    min-width: 10pc;
```
Comment (yndajas) [inline app/assets/stylesheets/components/_map.scss:143]: Can we do this with `rem` instead?

File: app/javascript/components/map/MapPopup.vue [hunk 2]
```diff
@@ -8,10 +8,6 @@
       <BarChart :chartData="data" :options="options" :key="randomKey" />
       <MapLegend :data="data.datasets" />
     </div>
-    <div class="map__info-box">
-      <span class="map__info-box-icon"></span>
-        {{ this.text }}
```
Comment (yndajas) [inline app/javascript/components/map/MapPopup.vue:13]: I think this can be removed from `data` and the `getChartData` method now

**Review by yndajas — CHANGES_REQUESTED** [review]
Looks good. Just a couple of small comments

File: app/assets/stylesheets/components/_map.scss (hunk 1 above)
Comment (sergiomarrocoli) [reply app/assets/stylesheets/components/_map.scss]: done

File: app/javascript/components/map/MapPopup.vue (hunk 2 above)
Comment (sergiomarrocoli) [reply app/javascript/components/map/MapPopup.vue:13]: done

**Review by yndajas — APPROVED** [review]

**Pull request merged**